app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -79,6 +79,32 @@ def metrics_connect
                                      authentication_password('default'))
     end
 
+    def run_generic_operation(operation, path)
+      require 'hawkular_all'
+      host_port = "#{hostname}:#{port}"
+      client = ::Hawkular::Operations::OperationsClient.new(:host        => host_port,
+                                                            :credentials => {
+                                                              :username => authentication_userid('default'),
+                                                              :password => authentication_password('default')
+                                                            })
```

Comment:
Lets please do that in a separate PR as there is also the metrics_connect to fix.
And in hawkular-client-gem 0.3.0 the whole connection setup will change anyway, as we will only have one entrypoint then and not per subsystem.